File: C/Ex15.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    float P, A, R;

    printf("Peso: ");
    scanf("%f", &P);

    printf("Altura: ");
    scanf("%f", &A);

    R = P/(A*A);

    if(R<20){
        printf("Abaixo do peso %f", R);
    }
    else if(20<R<25){
        printf("Peso ideal %f", R);
    }
    else{
        printf("Acima do peso %f", R);
    }
    return 0;
}

File: Python/Ex15.py
peso = float(input("Digite o peso: "))
altura = float(input("Digite a altura: "))

resultado = peso/(altura*altura)
 
if (resultado < 20):
    print("Abaixo do peso.")
elif(resultado >=25):
    print("Acima do peso.")
elif(resultado>=20 and resultado < 25):
    print("Peso ideal.")